tests/python/pytorch/dataloading2/test_csc_sampling_graph.py

```diff
@@ -0,0 +1,124 @@
+import unittest
+
+import backend as F
+
+import numpy as np
+import pytest
+import torch
+
+from dgl.dataloading2 import *
+
+
+def random_heterogeneous_graph_from_csc(num_nodes, num_ntypes, num_etypes):
```

Comment:
Please fix seeds for all random generators in test. Otherwise, later contributors may see random failures.